libs/payments/capability/src/lib/capability.manager.ts

```diff
@@ -14,11 +14,9 @@ export class CapabilityManager {
   constructor(private contentfulManager: ContentfulManager) {}
 
   async getClients() {
-    const clients: ServiceResult[] = (
-      await this.contentfulManager.getServicesWithCapabilities()
-    ).getServices();
-
-    if (!clients) return [];
+    const resultUtil =
+      await this.contentfulManager.getServicesWithCapabilities();
+    const clients = resultUtil.getServices();
```

Comment:
Not clear about this change, the Refactoring book would actually suggest this gets changed to what it was as it introduces placeholder variables which pollute local scope with more names.

Reply:
I generally am firmly against wrapping any await-ed call in a parenthesis because it’s difficult to read or understand. In this instance it gets formatted quite weirdly. Nonetheless, I'm happy to change it back - the point here was mainly to remove the typecast.